tests/dags/providers/provider_api_scripts/test_brooklyn_museum.py

```diff
@@ -55,6 +52,12 @@ def test_get_data_from_response(resource_name, expected):
     assert actual == expected
 
 
+def test_docstring_example():
+    json_func_ex = make_resource_json_func("brooklynmuseum")
+    json_dict = json_func_ex("cc_license_info.json")
+    assert json_dict["public_name"] == "Creative Commons-BY"
+
+
```

Comment:
Is this still necessary? Should we perhaps have a separate test file specifically for testing the `make_resourse_json_func` function?

Reply:
Yeah splitting out makes sense. I really only added this test because I found myself writing out the logic for a test as an example in the docstring, and then having an entire test that lives in a doc string and not the actual test suite didn't feel right.